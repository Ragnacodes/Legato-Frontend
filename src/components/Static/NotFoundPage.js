import React from 'react';
import { NavLink } from 'react-router-dom';
import { Button } from '@material-ui/core';
import PageNotFoundSVG from '../../styles/assets/page-not-found.svg';

const NotFoundPage = () => {
    return (
        <div className="centered-container">
            <img
                src={PageNotFoundSVG}
                width="400px"
                height="400px"
                alt="404 Page Not Found."
            />
            <Button component={NavLink} to="/" color="primary">
                Back To Home
            </Button>
        </div>
    );
};

export default NotFoundPage;