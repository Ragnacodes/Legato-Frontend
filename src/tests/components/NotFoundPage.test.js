import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/Layout/NotFoundPage';

test('should render NotFoundPage correctly', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
