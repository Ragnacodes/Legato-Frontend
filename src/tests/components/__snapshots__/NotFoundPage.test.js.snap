// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div
  className="centered-container"
>
  <img
    alt="404 Page Not Found."
    height="400px"
    src="page-not-found.svg"
    width="400px"
  />
  <WithStyles(ForwardRef(Button))
    color="primary"
    component={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "propTypes": Object {
          "activeClassName": [Function],
          "activeStyle": [Function],
          "aria-current": [Function],
          "className": [Function],
          "exact": [Function],
          "innerRef": [Function],
          "isActive": [Function],
          "location": [Function],
          "onClick": [Function],
          "replace": [Function],
          "sensitive": [Function],
          "strict": [Function],
          "style": [Function],
          "target": [Function],
          "to": [Function],
        },
        "render": [Function],
      }
    }
    to="/"
  >
    Back To Home
  </WithStyles(ForwardRef(Button))>
</div>
`;
